### PR TITLE
Adds a tool for importing .netset files into the ban DB.

### PR DESCRIPTION
**Pull request by moonheart08**
## About the PR
.netset is a stupidly simple format (each line is either a comment, or an IP) so I wrote a tool to import these files into the ban database. 

Also the banning user is MoMMI, which is a reserved/unusable account, so that should make it easier to sort automated bans from non-automated.

We can then use this to import https://github.com/firehol/blocklist-ipsets/blob/master/datacenters.netset later.

### CODE REVIEW

File: Content.Server/Administration/Commands/ImportNetsetCommand.cs [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+            var def = new ServerBanDef(
+                null,
+                null,
+                (ip, cidr),
+                null,
+                DateTimeOffset.UnixEpoch,
+                null,
+                args[1],
+                new NetUserId(new Guid("a17aa146-807c-48bb-8573-65e3cb05119d")),
```
Comment (wrexbe) [inline Content.Server/Administration/Commands/ImportNetsetCommand.cs:98]: If this is MoMMI's user, maybe this should be a CCVar, or a constant with a name.

**Review by wrexbe — CHANGES_REQUESTED** [review]
I feel like this command could be abused to anonymously ban people. Maybe force add who did it to the reason, or at log when someone uses this command.

**Comment by moonheart08** [comment]
> I feel like this command could be abused to anonymously ban people. Maybe force add the person who uses it to the ban reason, or log when someone uses this command.

This is an issue on the administrative side, there's a reason this tool is +HOST.

File: Content.Server/Administration/Commands/ImportNetsetCommand.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (moonheart08) [reply Content.Server/Administration/Commands/ImportNetsetCommand.cs:98]: I don't think this needs to be modifiable.

[the remark this answers is not in the document]
Comment (moonheart08) [reply Content.Server/Administration/Commands/ImportNetsetCommand.cs]: I've made it clearer what it is, however I want the user to remain consistent, ideally between servers, for automation purposes.

**Review by wrexbe — APPROVED** [review]

**Pull request merged**